### DISCUSSION TITLE
Facing an issue when I am using QInputDate and AgGrid.

### DISCUSSION THREAD

**Discussion by samuellleow** [elimintz/justpy#357] [General]
![image](https://user-images.githubusercontent.com/60358173/156491626-6d0dcf48-7b2e-4853-b54a-7d12b6c94ae7.png)

# Facing a bug/issue when I am using QInputDate and AgGrid.
![image](https://user-images.githubusercontent.com/60358173/156491982-3b6c7ab3-0003-4d35-8db2-321636f089cf.png)

After selecting a date using QInputDate and clicking on "Run", I will select records from a dummy dataframe that satisfy the selected date. I will then pass then dataframe (which does not contain NaN value) into 
`grid = jp.AgGrid(style='height: 600px; width: 100%; margin-top:1%')`
`grid.load_pandas_frame(df)`
However, the error above will be shown in console.

Current solution that I have found:
![image](https://user-images.githubusercontent.com/60358173/156493003-11ae9882-3134-41d2-8afe-5e6ecdb97d2d.png)
Adding line 19 into aggrid.js within justpy

**Comment by elimintz** [comment]
Thanks, will take a look and add this change
